cmd/mavenBuild.go

```diff
@@ -49,6 +50,23 @@ func runMavenBuild(config *mavenBuildOptions, telemetryData *telemetry.CustomDat
 		defines = append(defines, "-Dflatten.mode=resolveCiFriendliesOnly", "-DupdatePomFile=true")
 	}
 
+	commonPipelineEnvironment.custom.createBom = config.CreateBOM
```

Comment:
not sure if we should enhance CPE in such a way.
We know already today, that we have to call multiple build steps, e.g. `mavenBuild`, `kanikoExecute`, ...
Which of the `dockerImages` will be available? With the current approach it seems to be the last one ...